scripts/react-native-xcode.sh

```diff
@@ -56,11 +56,8 @@ case "$CONFIGURATION" in
     ;;
 esac
 
-# Path to react-native folder inside node_modules
-REACT_NATIVE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
```

Comment:
REACT_NATIVE_DIR is used just below to source `scripts/node-binary.sh`

Reply:
Thanks! 😓 I got a bit too aggressive on that one.